hollow/src/main/java/com/netflix/hollow/tools/history/keyindex/HollowHistoryTypeKeyIndex.java

```diff
@@ -213,6 +217,28 @@ public IntList queryIndexedFields(final String query) {
             return matchingKeys;
         }
 
+        String[] parts;
+        if (query.contains(":")) {  // composite field query, uses ':' as separator
```

Comment:
minor: it would be great to have constant for ":" as well

Reply:
done